Teaching/WebAssembly/webassembly.md

```diff
@@ -0,0 +1,31 @@
+## Background
+
+WebAssembly is a technology in web development growing in popularity for complex/time-consuming computational taks.  
```

Comment:
It would be good to explain a bit about what WebAssembly does: Compiled language for the browsers.